redisinsight/ui/src/pages/settings/components/cloud-settings/CloudSettings.tsx

```diff
@@ -58,19 +58,22 @@ const CloudSettings = () => {
       <Spacer size="s" />
       <Row gap="m" responsive>
         <FlexItem grow>
-          <Text size="s" className={styles.smallText} color="subdued">
-            The list of API user keys that are stored locally in Redis Insight.{' '}
-            <br />
-            API user keys grant programmatic access to Redis Cloud. <br />
-            {'To delete API keys from Redis Cloud, '}
+          <Text size="m" className={styles.smallText} color="primary">
+            The list of API user keys that are stored locally in Redis Insight.
+          </Text>
+          <Spacer size="xs" />
+          <Text size="m" className={styles.smallText} color="primary">
+            API user keys grant programmatic access to Redis Cloud.
+          </Text>
+          <Text size="m" className={styles.smallText} color="primary">
+            To delete API keys from Redis Cloud,
             <Link
               target="_blank"
-              color="text"
               href="https://redis.io/redis-enterprise-cloud/overview/?utm_source=redisinsight&utm_medium=settings&utm_campaign=clear_keys"
```

Comment:
Tip: It's not related anyhow to the visual changes you made, but if you want, you can move this URL to live next to the rest of the `EXTERNAL_LINKS` [here](https://github.com/redis/RedisInsight/blob/main/redisinsight/ui/src/constants/links.ts).

And then, you can use `getUtmExternalLink` to build up the URL with all the necessary query parameters, instead of having them hardcoded.

```suggestion
<Link
    color="primary"
    target="_blank"
    href={getUtmExternalLink(EXTERNAL_LINKS.redisEnterpriseCloud, {
        medium: UTM_MEDIUMS.Settings,
        campaign: 'clear_keys',
    })}}
>
    Privacy Policy
</Link>
```

Reply:
yes, should have changed that based on our discussion in my previous pr... tnx! will change it in a bit